ci/test_python.sh

```diff
@@ -88,6 +89,23 @@ pytest \
   cugraph/pytest-based/bench_algos.py
 popd
 
+rapids-logger "pytest cugraph-nx"
+pushd python/cugraph-nx/cugraph_nx
+pytest \
+  --capture=no \
+  --verbose \
+  --cache-clear \
+  --junitxml="${RAPIDS_TESTS_DIR}/junit-cugraph-service.xml" \
+  --cov-config=../../.coveragerc \
+  --cov=cugraph_nx \
+  --cov-report=xml:"${RAPIDS_COVERAGE_DIR}/cugraph-nx-coverage.xml" \
+  --cov-report=term \
+  --benchmark-disable \
+  tests
+popd
+
+# FIXME: add call to test cugraph-nx against the networkx test suite
```

Comment:
Hopefully this FIXME is easily resolved once #3808 is merged by running `run_nx_tests.sh` with the necessary extra pytest arguments.

Reply:
A call to `run_nx_tests.sh` has been added now.